blackbox/docs/sql/statements/set-license.rst

```diff
@@ -0,0 +1,60 @@
+.. highlight:: psql
+.. _ref-set-license:
+
+===============
+``SET LICENSE``
+===============
+
+Sets the license Key.
+
+.. rubric:: Table of Contents
+
+.. contents::
+    :local:
+
+Synopsis
+========
+
+::
+
+    SET LICENSE license_key
+
+Description
+===========
+
+``SET LICENSE`` registers a CrateDB license key to the cluster.
+If the license registration is successful, CrateDB will continue to operate
+in `Enterprise Edition`_ until the license expiry date.
```

Comment:
Maybe we should mention here that a switch to community edition is possible - with a link to the enterprise setting?